README.md

```diff
@@ -118,22 +118,41 @@ The analyze mode emits a list of capabilities often seen in malware, categorized
 Requirements:
 
 * [go](https://go.dev/) - the programming language
+* [rust](https://www.rust-lang.org) - yara-x requirement
 * [yara-x](https://virustotal.github.io/yara-x/) - the rule language
```

Comment:
yara-x isn't technically a rule language, it's a rust implementation of the YARA rule language.

Reply:
Updated in [`d39a001` (#835)](https://github.com/chainguard-dev/malcontent/pull/835/commits/d39a00102517a38d70952af23d6d0fc41c86ce3f).